Projects/UOContent/Gumps/RunebookGump.cs

```diff
@@ -7,504 +7,448 @@
 using Server.Spells.Fourth;
 using Server.Spells.Seventh;
 
-namespace Server.Gumps
-{
-    public class RunebookGump : Gump
-    {
-        public RunebookGump(Mobile from, Runebook book) : base(150, 200)
-        {
-            Book = book;
-
-            AddBackground();
-            AddIndex();
-
-            for (var page = 0; page < 8; ++page)
-            {
-                AddPage(2 + page);
+namespace Server.Gumps;
 
-                AddButton(125, 14, 2205, 2205, 0, GumpButtonType.Page, 1 + page);
+public class RunebookGump : DynamicGump
+{
+    private readonly Runebook _book;
+    public Runebook Book => _book;
```

Comment:
[See this linky link for details on Book](https://github.com/modernuo/ModernUO/pull/1915/files#diff-b59c51b0cd5556d3336f71385690c4b319a649247b39de94ce3a3dac1703a6a5R184)